testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -452,6 +471,9 @@ public void testSelectInformationSchemaColumns()
     @Test
     public void testShowCreateTable()
     {
+        if (!hasBehavior(SUPPORTS_SCALAR_QUERIES)) {
+            return;
+        }
```

Comment:
Revert.

Reply:
In the case of Pinot, if these queries are not supported, what is expected to be taken into account? Pinot queries fail with some of these and so, I had introduced the `hasBehavior` for tests which Pinot can ignore.

Reply:
Please see other comment https://github.com/trinodb/trino/pull/13619#discussion_r950909571